spec/index.html

```diff
@@ -1362,6 +1367,16 @@ <h2>Internationalization Considerations</h2>
   <p class="issue" data-number="9"></p>
 </section>
 
+<section id="iri-abnf" class="appendix informative">
+  <h2>IRI Grammar</h2>
+
+  <p>The following [[ABNF]] grammar is from [[RFC3987]] and includes the referenced
+    productions from [[RFC3986]].</p>
```

Comment:
```suggestion
  <p>
    The following [[ABNF]] grammar applies the changes from [[RFC3987]] to the section
    <a href="https://www.rfc-editor.org/rfc/rfc3986#appendix-A">Collected ABNF for URI</a> of
    [[RFC3986]] to give a consolidated grammar for IRIs.
   </p>
   <p>
    This is provided for convenience only.
    If it differs from definitions in [[RFC3986]], [[RFC3987]] or any subsequent updates,
    then those definitions should be used.
    </p>
  ```

Reply:
Words to say that the RFCs are definitive/normative.

Reply:
Updated to use `data-cite`.